README.rst

```diff
@@ -0,0 +1,40 @@
+FogLAMP "random" C++ South plugin
+
+Build
+----
+To build FogLAMP Random C++ South plugin:
+- mkdir build
+- cd build
+- cmake ..
+
+- By default the FogLAMP develop package header files an libraries
+  are in use: /usr/include/foglamp and /usr/lib/foglamp
+- If FOGLAMP_ROOT env var is set and no -D options are set,
+  the header files and libraries paths are set to FOGLAMP_ROOT dir
+
+Options to pass to cmake:
```

Comment:
"Options to pass to cmake" -> "You may also pass one or more of the following options to cmake to override this default behaviour."